tests/integration/012_environment/main.tf

```diff
@@ -19,6 +19,14 @@ resource "env0_template" "template" {
   terraform_version = "0.15.1"
 }
 
+resource "env0_template" "workflow_subenv_null_resource" {
+  name              = "workflow-subenv-null-resource"
```

Comment:
please add a `random` value to the name, to prevent flakiness issues, like if the destruction fails we can't create another template with the same name.

Reply:
we can't do it here. the subenvironment template is hardcoded in the workflow file. can you suggest a workaround for that? we can also remove this test while we figuring our a solution

Reply:
I'm going into an interview... won't be available RN

Reply:
Ok, I see that it works since #579, let's keep it in meanwhile...